### PR TITLE
Use heartbeat event to send metrics

### PR DESCRIPTION
**Pull request by volodymyr-mykhailyk**
This change simplifies dependencies and using internal sidekiq mechanism to send all required metrics to datadog.

### CODE REVIEW

**Comment by volodymyr-mykhailyk** [comment]
In subsequent PRs I will extend list of metrics we are sending

File: README.md [hunk 1]
```diff
@@ -25,24 +25,23 @@ To start sending metrics
 
 ```ruby
 # Import the library
-require 'sidekiq/datadog/monitor/data'
+require 'sidekiq/datadog/monitor'
 
 # Initiate a Sidekiq::Datadog::Monitor client instance.
-Sidekiq::Datadog::Monitor::Data.initialize!(
+Sidekiq::Datadog::Monitor.initialize!(
```
Comment (omaik) [inline README.md:31]: guess `configure!`

File: lib/sidekiq/datadog/monitor.rb [hunk 2]
```diff
@@ -1,12 +1,68 @@
...
+          agent_host && agent_port
+        end
+
+        def initialize!
+          @statsd = ::Datadog::Statsd.new(agent_host, agent_port)
+          @sender = Sidekiq::Datadog::Monitor::MetricsSender.new(statsd, tags)
+        end
+
+        def send_metrics
+          raise Sidekiq::Datadog::Monitor::Error, "Can't send metrics, gem not initialized" unless sender
```
Comment (omaik) [inline lib/sidekiq/datadog/monitor.rb:33]: how it is possible, heartbeat may be invoked before startup ?

**Review by omaik — APPROVED** [review]

File: lib/sidekiq/datadog/monitor.rb (hunk 2 above)
Comment (volodymyr-mykhailyk) [reply lib/sidekiq/datadog/monitor.rb]: Probably not, sometimes I get sidetracked and overdo someof the checks. Removed this.

File: README.md (hunk 1 above)
Comment (volodymyr-mykhailyk) [reply README.md]: Fixed

**Review by omaik — APPROVED** [review]

**Review by danpolyuha — APPROVED** [review]
👍

**Pull request merged**